lib/Pill/index.js

```diff
@@ -3,12 +3,15 @@ import { string } from 'prop-types';
 import { types, defaults } from './pillTypes';
 import PillDefault from './PillDefault';
 import PillError from './PillError';
+import PillNew from './PillNew';
 
 const Pill = ({ type, children, ...rest }) => {
   const getPill = () => {
     switch (type) {
       case 'error':
         return PillError;
+      case 'new':
+        return PillNew;
```

Comment:
🔴 I'm not a huge fan of the "new" name here. Might be worth going back to the drawing board here.

I wonder if we could hook into primary, secondary etc here? The button updates have shown how sweet that has been so maybe we could carry the learnings from that?